client-js/package.json

```diff
@@ -13,23 +13,24 @@
     "clean": "rm -rf ./dist",
     "prebuild": "npm run clean",
     "build": "tsc --project tsconfig.build.json",
-    "generate:wasm": "ts-node \"src/scripts/generate-wasm.ts\""
+    "generate:wasm": "ts-node \"src/scripts/generate-wasm.ts\"",
+    "postinstall": "npm run generate:wasm"
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/casper-ecosystem/erc20.git"
+    "url": "git+https://github.com/casper-ecosystem/cep18.git"
   },
   "keywords": [
     "casper",
-    "erc20",
+    "cep18",
     "fungible token"
   ],
   "author": "Ryo Kanazawa <ryo@casperlabs.io>",
   "license": "Apache License 2.0",
   "bugs": {
-    "url": "https://github.com/casper-ecosystem/erc20/issues"
+    "url": "https://github.com/casper-ecosystem/cep18/issues"
   },
-  "homepage": "https://github.com/casper-ecosystem/erc20#readme",
+  "homepage": "https://github.com/casper-ecosystem/cep18#readme",
```

Comment:
change to `https://github.com/casper-ecosystem/cep18/tree/master/client-js#readme` :)